web/app/charts/SankeyEditor.js

```diff
@@ -352,9 +437,12 @@ bluewave.charts.SankeyEditor = function(parent, config) {
           //Update nodes
             var node = nodes[inputID];
             node.inputs[outputID] = nodes[outputID];
-
+            var nodeOut = nodes[outputID];
+            nodeOut.outputs[inputID] = nodes[inputID];

```

Comment:
Can we try to avoid using outputs? I think we can rely on inputs alone. It will make our lives a little easier when importing/exporting the sankey config

Reply:
We no longer use outputs, just inputs now.